packages/core/lib/config/mainnet/plugins.js

```diff
@@ -43,8 +43,8 @@ module.exports = {
       host: process.env.ARK_DB_HOST || 'localhost',
       port: process.env.ARK_DB_PORT || 5432,
       database: process.env.ARK_DB_DATABASE || `ark_${process.env.ARK_NETWORK_NAME}`,
-      user: process.env.ARK_DB_USERNAME || 'ark',
-      password: process.env.ARK_DB_PASSWORD || 'password'
+      user: process.env.ARK_DB_USERNAME || 'fixcrypt',
```

Comment:
Those changes shouldn't be committed as they are your personal configuration.

Reply:
yes somehow that file slipped, pushing a revert